Product/Production/Common/Properties/src/main/resources/CORE_X12DSGenericBatchProxyConfig.xml

```diff
@@ -12,8 +12,7 @@ http://www.springframework.org/schema/aop     http://www.springframework.org/sch
         The beans available in this file must be identified in the "description" element for the Spring configuration application. Beans are listed between braces
```

Comment:
It probably should be -- I ran more builds for this ticket alone than I would for two normal sprints combined, so I got sidetracked by some of the build warnings I saw.  Since that cleanup item doesn't affect the POMs, I am good with splitting it out into a new ticket if needed.